.github/workflows/deploy.yml

```diff
@@ -42,91 +40,84 @@ jobs:
         with:
           ref: ${{ github.head_ref }}
           fetch-depth: 0
-
       - name: Install terraform
-        run: |
-          if ! which terraform 2>/dev/null; then
-            sh tools/install/terraform.sh
-          fi   
-
+        run: sh tools/install/terraform.sh
       - name: Check Format
         run: terraform fmt -check -recursive -diff
-
-  AdminGUI:
+  Deployment :
     runs-on: ubuntu-latest
     steps:
-      - name: Get type of VM
-        run: |
-          lscpu
-          uname -a
-          grep MemTotal /proc/meminfo
-
       - name: Checkout
         uses: actions/checkout@v3
         with:
-          ref: ${{ github.head_ref }}
-          fetch-depth: 0
-
+           ref: ${{ github.head_ref }}
+           fetch-depth: 0
+      - name: Install Dependencies
+        uses: aneoconsulting/ArmoniK.Action.Deploy/dependencies@main
+        with:
+           docker: true
+           terraform: true
+           k3s: true
+      - name: Deploy
+        uses: aneoconsulting/ArmoniK.Action.Deploy/deploy@main
+        with:
+          type: localhost
+          ext-csharp-version: ${{ steps.worker.outputs.tag }} 
       - name: Install AWS CLI
+        run: sh tools/install/aws-cli.sh
```

Comment:
If this script is still needed, it should be part of the action. But I think you can just remove it (and add `aws=true` when calling the dependencies action)